templates/github-actions/scan-pull-requests/frogbot-scan-pull-requests.yml

```diff
@@ -0,0 +1,79 @@
+name: "Frogbot Scan Pull Requests"
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+permissions:
+  pull-requests: write
+  contents: write
+jobs:
+  scan-pull-requests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Install prerequisites
+
+      # Uncomment if you have a Pip/Pipenv/Poetry project
+      # - uses: actions/setup-python@v4
+      #   with:
+      #     python-version: "3.x"
+
+      # Uncomment if you have Pipenv project
+      # - name: Install poetry
+      #   run: pip3 install poetry
+
+      # Uncomment if you have Poetry project
+      # - name: Setup Poetry
+      #   run: pip3 install poetry
+
+      # Uncomment if you have NuGet project
+      # - uses: nuget/setup-nuget@v1
+      #   with:
+      #     nuget-version: "5.x"
+
+      # Uncomment if you have .NET project
+      # - uses: actions/setup-dotnet@v2
+      #   with:
+      #     dotnet-version: "3.1.x"
+
+      # Uncomment if you have Maven/Gradle project
+      # - name: Set up Java
+      #   uses: actions/setup-java@v3
+      #   with:
+      #     java-version: "11"
+      #     distribution: "temurin"
+
+      # Uncomment if you have Gradle project
+      # - name: Setup Gradle
+      #   uses: gradle/gradle-build-action@v2
+
```

Comment:
I think that Go, npm, and Yarn are missing

Reply:
done